examples/XmlExample.php

```diff
@@ -1,22 +1,14 @@
 <?php
 
-require_once '../vendor/autoload.php';
+require_once __DIR__ . '/../vendor/autoload.php';
 
+use FINDOLOGIC\Export\Data\Item;
```

Comment:
Please remove the backslashes of the other use statements for consistancy.

Reply:
✅